app/actions.py

```diff
@@ -231,6 +235,21 @@ def bookmarkRemove(self):
       (boolean) Whether any bookmarks were removed.
     """
     upperRow, _, lowerRow, _ = self.startAndEnd()
+    return self.bookmarkRemoveFrom(upperRow, lowerRow)
+
+  def bookmarkRemoveFrom(self, upperRow, lowerRow):
+    """
+    Removes bookmarks in the given line interval.
+
+    Args:
+      upperRow: The upper line limit.
+      lowerRow: The lower line limit.
+
+    Returns:
+      (boolean) Whether any bookmarks were removed.
+    """
+    if upperRow > lowerRow:
+      return
```

Comment:
Even though None will equate to False, let's throw a False on this return.

Reply:
Sorry, totally forgot about the return value.